tf2_ros/src/tf2_echo.cpp

```diff
@@ -125,6 +128,7 @@ int main(int argc, char ** argv)
       source_frameid.c_str(), target_frameid.c_str(), warning_msg.c_str());
     rate.sleep();
   }
+  constexpr double rad_to_deg = 180.0 / 3.14159265358979323846;
```

Comment:
Yes, the define has to be *before* `#include <cmath>`.  In practical terms, it means it has to be the very first non-comment thing in the file (since you don't know which headers will pull in cmath recursively).

Reply:
I've now fixed this in the latest code.